bol/ajax_service.php

```diff
@@ -586,14 +586,26 @@ public function findUsers( $kw, $limit = null )
             "method" => "MAILBOX_BOL_AjaxService::findUsers"
         ));
 
+        $params = array('kw' => $kw . '%');
+
+        if ( $kw !== null )
+        {
+            if ( $questionName == "username" )
+            {
+                $queryParts["where"] .= " AND base_user_table_alias.username LIKE :kw";
```

Comment:
Do not use "base_user_table_alias" alias - it is private alias for getUserQueryFilter.